examples/tutorials/advanced/date_time_charts.py

```diff
@@ -193,9 +193,9 @@
 # initialized as a list of :class:`xarray.DataArray` objects. This object
 # provides a wrapper around regular PyData formats. It also allows the data to
 # have labeled dimensions while supporting operations that use various pieces
-# of metadata.The following code uses :func:`pandas.date_range` object to fill
-# the DataArray with data, but this is not essential for the creation of a
-# valid DataArray.
+# of metadata. The following code uses :func:`pandas.date_range` object to
```

Comment:
```suggestion
# of metadata. The following code uses a :func:`pandas.date_range` object to
```

Reply:
Thanks @michaelgrund for mentioning this missing word! I looked again at this tutorial and found more typos. Thus, I submitted a separate PR #2669.